.travis.yml

```diff
@@ -214,10 +215,21 @@ script:
   #
   # Enable COREYX (swapped)
   #
-  #- restore_configs
-  #- opt_enable COREYX
-  #- build_marlin
+  - restore_configs
+  - opt_enable COREYX
+  - build_marlin
   #
+  # Test Endstop Interrupts Feature
+  #
+  - restore_configs
+  - opt_enable ENDSTOP_INTERRUPTS_FEATURE
+  - opt_enable USE_XMIN_PLUG
+  - opt_enable USE_XMAX_PLUG
+  - opt_enable USE_YMIN_PLUG
+  - opt_enable USE_YMAX_PLUG
+  - opt_enable USE_ZMIN_PLUG
+  - opt_enable USE_ZMAX_PLUG
+  - build_marlin
```

Comment:
Please try to combine as many tests as possible into a single build.

Reply:
I will do that.